net/Socket.cpp

```diff
@@ -1145,63 +1158,160 @@ bool ServerSocket::bind([[maybe_unused]] Type type, [[maybe_unused]] int port)
 #endif
 }
 
+bool ServerSocket::isRetryAcceptError(const int cause)
```

Comment:
Really not a fan of this huge list of extremely marginal and/or unusual errors. Previously we'd fail hard in these cases - lets handle EINTR/EAGAIN/EWOULDBLOCK one way, and EMFILE the other in-line, and ERR_SYS and continue with everythign else (?).

Reply:
Yes, it is huge and determination of recoverable errors depends on whole system esp. for EMFILE/ENFILE and the other 'out of memory' symbols.
The latter memory causes may also be candidates to influence the discussed finer grained limitation behavior at a later stage.
Will refine it.